packages/app/components/drop/drop-event.tsx

```diff
@@ -554,21 +554,17 @@ export const DropEvent = () => {
                     </View>
                     <ScrollView tw="flex-row" horizontal={true}>
                       <DataPill
-                        label={`Royalties ${watch("royalty")}%`}
+                        label={`${watch("editionSize")} Editions`}
```

Comment:
This does not take into account when there is 1 Edition, 2 Edition**s**
Singular/Plural

Reply:
👍 yeah, I was though it too.

Reply:
just fixed and update it, thanks remind me.